numba/cuda/tests/cudadrv/test_host_alloc.py

```diff
@@ -45,6 +43,18 @@ def test_host_alloc_mapped(self):
         self.assertTrue(all(ary == 123))
         driver.device_memset(ary, 0, driver.device_memory_size(ary))
         self.assertTrue(all(ary == 0))
+        self.assertTrue(sum(ary != 0) == 0)
```

Comment:
Thanks for adding these extra checks - I think it would be good if these additions could be moved into a separate test case `test_host_operators`, perhaps) so that we don't have one test case testing too many things (at the moment in here there's allocation, memset, and the operator tests) and also test these with the pinned arrays (I think these should all work for both pinned and mapped arrays?).